test/audits/image-without-alt-text-test.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function() {
+    module('ImageWithoutAltText');
+    var rule = axs.AuditRules.getRule('imagesWithoutAltText');
+
+    test('Image with no text alternative', function() {
+        var fixture = document.getElementById('qunit-fixture');
+        var img = fixture.appendChild(document.createElement('img'));
+        img.src = 'smile.jpg';
+        var expected = { elements: [img], result: axs.constants.AuditResult.FAIL };
+        deepEqual(rule.run({ scope: fixture }), expected, 'Image has no text alternative');
+    });
+
```

Comment:
I think two more are needed for completeness to check that presentational images are handled. One with null alt text and one with role presentation.